style.css

```diff
@@ -156,3 +156,39 @@ a {
 	text-underline-offset: .15em
 }
 
+
+/* Reduce the default vertical distance between elements when inside a layout flow */
+body .is-layout-flow > * + * {
+    margin-block-start: 0.5rem;
+}
+
+/* Increase the default vertical distance between elements when inside a layout flow */
+body .is-layout-flow .wp-block-group {
+   margin-block-end: 5rem;
+}
+
+/*
+ * Search styles
+ */
+.wp-block[data-type="core/search"] {
+	margin-bottom: 5rem !important;
```

Comment:
can we remove the **!important** here and handle the margin-bottom  in different way?

Reply:
Fixed here [91c65d7](https://github.com/Automattic/sensei-theme/pull/36/commits/91c65d760673ce5c07ca5034d273ed85f4283c34)